src/java/org/apache/cassandra/security/PEMBasedSslContextFactory.java

```diff
@@ -92,33 +92,45 @@ public final class PEMBasedSslContextFactory extends FileBasedSslContextFactory
     private static final Logger logger = LoggerFactory.getLogger(PEMBasedSslContextFactory.class);
     private String pemEncodedKey;
     private String keyPassword;
+    private String pemEncodedOutboundKey;
+    private String outboundKeyPassword;
```

Comment:
Could you have created a PEMBasedKeyStoreContext to hold this in? Then you could avoid duplication in `buildKeyManagerFactory` and `buildOutboundKeyManagerFactory`